components/dashboard/SentimentTracker/SentimentTracker.js

```diff
@@ -160,7 +162,7 @@ const SentimentTracker = props => {
 SentimentTracker.propTypes = {
   onRecord: PropTypes.func,
   onClose: PropTypes.func.isRequired,
-  user: PropTypes.string.isRequired,
+  record: PropTypes.string.isRequired,
```

Comment:
It's unclear at a glance what this variable name means; could we please change to something more descriptive (e.g. `lastRecordedValue`)?

Reply:
totally agree.